src/pages/login/index.page.tsx

```diff
@@ -71,38 +76,14 @@ function Login() {
             id="password"
             label="비밀번호"
             hasLabel={true}
-            icon={{
-              src: '/svgs/search.svg',
-              alt: 'search 아이콘',
-              width: 24,
-              height: 24,
-            }}
             type="password"
-            required
             register={register('password')}
             placeholder="비밀번호를 입력해주세요."
+            errors={errors}
           />
-          <p>{errors.password?.message}</p>
-        </div>
-        <div>
-          <Input
-            id="passwordConfirm"
-            type="password"
-            label="비밀번호 확인"
-            hasLabel={false}
-            register={register('passwordConfirm')}
-            placeholder="비밀번호를 입력해 주세요."
-          />
-          <p>{errors.passwordConfirm?.message}</p>
         </div>
       </div>
-      <DatePicker
-        control={control}
-        name="ReactDatepicker"
-        label="마감일"
-        hasLabel={true}
-        required
-      />
+      <button type="submit">로그인</button>
```

Comment:
```suggestion
      <button type="submit">로그인</button>
```
type 명시 너무 좋네요!! 나중에 승구님이 만들어주신 버튼 컴포넌트로 바꿔주기만 하면 될 것 같아요